### DISCUSSION TITLE
Run "chia plots check" on the harvester to report an error

### DISCUSSION THREAD

**Discussion by longren610** [Chia-Network/chia-blockchain#7692] [Q&A] [answered]
I want to check whether the plot on the harvester is normal, so I execute "chia plots check" on the harvester, but the pk error will be reported during the execution. How should I solve this problem?
The error is as follows:
```
Plot E:\chiaplots\plot-k32-2021-06-11-08-31-65b0a2447c2*****************************c24b7a065.plot has a farmer public key that is not in the farmer's pk list.
```

**Comment by keliew** [comment]
You won't be able to farm that plot since you don't have the farmer's key. On the GUI, it would show too.

**Reply by longren610** [reply]
I operate on the harvester. According to the official statement, it is not safe for the harvester to directly add the key of the master node, so the master node key is not added to the harvester.

**Comment by rmica** [comment, marked as answer]
It's a warning not an error, he can't find the pk on the harvester because you don't have one. The warning applies to monolitic configurations.

**Reply by longren610** [reply]
Does it mean that the plot cannot be verified on the harvester?

**Reply by keliew** [reply]
yeah, can't be verified as far as i know.

you could add the farmer's key and run it, and then remove it.

**Reply by longren610** [reply]
Okay, thank you.

**Reply by rmica** [reply]
I don't think so. The plot can be verified without a pk, or better without the origin sk. The sk of the farmer must match with the pk of the plot only to sign the proof when farming. I don't think that plots check verifies the sign process. You don't need the sk in order to find a proof, you need it to sign.
To put it simple: the warning is telling you something you already knew, because you removed the keys from the harvester. It's relevant only in single-box configurations where you supposed to have the keys.

**Reply by longren610** [reply]
> I don't think so. The plot can be verified without a pk, or better without the origin sk. The sk of the farmer must match with the pk of the plot only to sign the proof when farming. I don't think that plots check verifies the sign process. You don't need the sk in order to find a proof, you need it to sign.
> To put it simple: the warning is telling you something you already knew, because you removed the keys from the harvester. It's relevant only in single-box configurations where you supposed to have the keys.

Yes, you are right. This warning is just a reminder. Just ignore it and wait for the command to complete. Thanks!
